### ISSUE TITLE
Startup crash

### ISSUE THREAD

**Issue by jaragunde** [Igalia/chromium#235]
Sometimes I get a crash like below on startup when running this fork on a Snapdragon 600 powered board. The browser stays still in the command line for some seconds (no window is shown), then the segfault message appears:
```
# /usr/lib/chromium/chromium --show-fps-counter --mus --ozone-platform=wayland --no-sandbox --user-data-dir=/tmp/chromium-data-dir --start-maximized  https://threejs.org/examples/webgl_geometries.html
[1427:1453:0913/194351.217122:ERROR:wayland_keyboard.cc(140)] Not implemented reached in static void ui::WaylandKeyboard::RepeatInfo(void*, wl_keyboard*, int32_t, int32_t)
libEGL warning: MESA-LOADER: failed to retrieve device information

MESA-LOADER: failed to retrieve device information
MESA-LOADER: failed to retrieve device information
[1427:1427:0913/194352.329672:ERROR:cursor_factory_ozone.cc(41)] Not implemented reached in virtual void* ui::CursorFactoryOzone::GetDefaultCursor(ui::CursorType)
[1427:1453:0913/194352.331116:ERROR:wayland_window.cc(288)] Not implemented reached in virtual void ui::WaylandWindow::SetCursor(ui::PlatformCursor)
[1427:1427:0913/194352.601535:ERROR:layer_tree_host_impl.cc(2404)] Forcing zero-copy tile initialization as worker context is missing
Segmentation fault (core dumped)
```

Oddly enough, it happens more often on first boot.

**Comment by r1mikey** [comment]
I've seen this on an x86-64 board today as well.  It seems to happen 1 out of every 3 or 4 times I start.

**Comment by tonikitoo** [comment]
are you able to produce debug builds, Michael?

**Comment by msisov** [comment]
It would be good to have core dumps at least if you are unable to get a build with symbols.

**Comment by r1mikey** [comment]
I'll see if I can get a debug build up.  My board is a network booted into RAM only, so it might be a bit tight.

**Comment by msisov** [comment]
Issue #237 might fix this problem. I faced it on aarch64 board today as well and tracked down that a bitmap update was called too early. Let's hope tonikitoo's patch resolves this for you, guys!

**Comment by r1mikey** [comment]
I'll give the patch a try, thanks!

I've really struggled to get a meaningful backtrace out of Chromium on my board.  What I've got keeps crashing the debugger!

**Comment by msisov** [comment]
@r1mikey , any updates?

**Comment by r1mikey** [comment]
Looks like my code drop (the most recent meta-browser release) is not sufficiently up to date to even apply the patch.

**Comment by tonikitoo** [comment]
awaiting for @jaragunde @fred-wang confirmations too, as they seems to have faced incarnations of this issue too (before the fix).

**Comment by tonikitoo** [comment]
Should be fixed by PR #237. Closing.

Please reopening if it is happens in latest 'ozone-wayland-dev' branch.
